interceptor/main.go

```diff
@@ -223,6 +223,7 @@ func runAdminServer(
 		},
 	)
 	kedahttp.AddConfigEndpoint(lggr, adminServer, servingConfig, timeoutConfig)
+	kedahttp.AddVersionEndpoint(lggr, adminServer)
```

Comment:
thoughts on the below?

```suggestion
	kedahttp.AddVersionEndpoint(lggr.WithName("interceptorAdmin"), adminServer)
```